toolchain/check/impl.cpp

```diff
@@ -119,8 +119,7 @@ auto ImplWitnessStartDefinition(Context& context, SemIR::Impl& impl) -> void {
   // witness.
   for (auto [assoc_entity, witness_value] :
        llvm::zip(assoc_entities, witness_block)) {
-    auto decl_id = assoc_entity;
-    decl_id = context.constant_values().GetConstantInstId(decl_id);
+    auto decl_id = context.constant_values().GetConstantInstId(assoc_entity);
```

Comment:
Are the changes in this file intended to be part of this PR? They look unrelated to me (missed file in #5185?).

Reply:
Sorry, wrong PR